src/DiagramGenerator/Config.php

```diff
@@ -64,15 +64,6 @@ class Config
     protected $pieceIndex;
 
     /**
-     * @Type("string")
-     * @StringOrInteger(min=0, max=3)
-     * @SerializedName("board")
-     * @var integer
-     */
-    // [lackovic10]: these are board texture folder names from the image url
-    protected $boardIndex;
```

Comment:
What happens to URLs with this parameter format? I assume the old site will continue to use the old DiagramGenerator, and then the new site will have a URL mapper of some sort?